test/functional/finalization_vote.py

```diff
@@ -147,6 +149,34 @@ def run_test(self):
                                          'validators': 3})
         self.log.info('Finalizers voted after configured block number')
 
+        generate_block(node0, count=4)
+        prev_tx = finalizer1.decoderawtransaction(prev_tx)['txid']
+
+        # test that node recognizes old and invalid votes.
+        tx = make_vote_tx(finalizer1, address1, node0.getblockhash(40), 5, 8, prev_tx)
+        assert_raises_rpc_error(-26, 'bad-vote-invalid', node0.sendrawtransaction, tx)
+        tx = make_vote_tx(finalizer1, address1, node0.getblockhash(40), 7, 9, prev_tx)
+        assert_raises_rpc_error(-26, 'bad-vote-invalid', node0.sendrawtransaction, tx)
+        tx = make_vote_tx(finalizer1, address1, node0.getblockhash(40), 5, 6, prev_tx)
+        assert_raises_rpc_error(-26, 'bad-vote-invalid', node0.sendrawtransaction, tx)
+        tx = make_vote_tx(finalizer1, address1, node0.getblockhash(40), 7, 6, prev_tx)
+        assert_raises_rpc_error(-26, 'bad-vote-invalid', node0.sendrawtransaction, tx)
+        self.log.info('Bad configured votes cannot screw up things')
```

Comment:
I would rather use `Tested outdated and invalid votes`, `screw up` is not exactly nice language 😅

Reply:
Fixed that!